sigle/src/modules/analytics/stats/StatsFrame.tsx

```diff
@@ -144,16 +99,15 @@ export const StatsFrame = () => {
             <TabsTrigger value="monthly">Monthly</TabsTrigger>
             <TabsTrigger value="all">All</TabsTrigger>
           </TabsList>
-          <TabsContent value="weekly"></TabsContent>
           <Box
             css={{
-              mb: '$5',
+              mb: '$8',
               position: 'relative',
               width: '100%',
               height: 400,
             }}
           >
-            <StatsChart type={statType} data={data} />
+            <StatsChart type={statType} data={data!} />
```

Comment:
let's try to not use `!` here, if there is no data then the graph should render empty correct? Maybe better to say that StatsChart data prop is not required?

Reply:
with this one, because there's the initial range already added as a placeholder, my assumption was that there should be always be data - is it because the data from the API could be returned undefined? If so I'm wondering the best place to have the conditional logic, as at this level it also removes the axis

Reply:
The API should not return undefined (unless there is a fetch error ofc), `data` is undefined on the first render in any case right?